app/templates/sites/companies/company/edit.hbs

```diff
@@ -0,0 +1,14 @@
+<h2>Edit Company</h2>
+
+<div class='form-group'>
```

Comment:
If you refactor this form into a component, you'll have better encapsulation of concerns related to this form. It will allow you to add logical error handing and you can even do test first by doing integration testing on the form.